cmd/observer/node_builder/observer_builder.go

```diff
@@ -106,7 +108,6 @@ type ObserverServiceConfig struct {
 	apiBurstlimits            map[string]int
 	rpcConf                   rpc.Config
 	rpcMetricsEnabled         bool
-	apiTimeout                time.Duration
```

Comment:
for observers, let's keep this timeout since it's specific for `upstream`, and just use it in place of `CollectionClientTimeout` and `ExecutionClientTimeout` in the connection factory